test/controllers/api/v2/hosts_bulk_actions_controller_test.rb

```diff
@@ -2,7 +2,13 @@
 
 require "katello_test_helper"
 
+unless Katello.with_remote_execution?
```

Comment:
This controller test doesn't invoke any code which references JobInvocationComposer, so this can be removed. It should be moved to the new test file for HostTracesManager. On current code, we have remote execution as a gem dependency, so it will always be defined. Although this code will be cherry-picked to an older branch which doesn't have that dep, so better to preserve it (in host traces manager test).

Reply:
Removed from both controller tests and added to the service class test.